tests/forms_tests/tests/tests.py

```diff
@@ -167,6 +167,10 @@ def test_initial_instance_value(self):
 
 
 class FormsModelTestCase(TestCase):
+    def test_modelchoicefield_manager(self):
```

Comment:
I wasn't sure where to put this test case, the class names in test files are confusing for me. Is this OK or can you suggest a better place?